website/docs/tutorials/08-cairo-1-support/01-understanding-cairo-packages.md

```diff
@@ -0,0 +1,144 @@
+# Understanding Cairo Packages
+
+There are several requirements that Cairo packages have to follow. Further documentation pages assume some knowledge
+of Cairo package structure. Please refer to this page for details.
+
+## `.cairo` files
+
+Files with `.cairo` extension contain cairo code, including Starknet contracts. A file can define multiple methods,
+structures etc.
+
+## Modules
+
+Module consists of one or more cairo files, usually organized in a single directory. To define a module, create `.cairo`
+file named like the module and define components of this module with `mod` keyword.
+
+```
+my_module/
+├── a.cairo
+├── b.cairo
+└── c.cairo
+my_module.cairo
+```
+
+```cairo title="my_module.cairo"
+mod a;
+mod b;
+mod c;
+```
+
+Alternatively, modules can be defined within a file using
+
+```cairo title="my_module.cairo"
+mod my_module {
+    fn function_a() -> ();
+    // ...
+}
+```
+
+## Packages
+
+Package is a collection of modules that also defines `cairo_project.toml` and `lib.cairo` files.
+
+:::info
+Some other tools use name "crates" for packages.
+:::
```

Comment:
The problem is, currently official cairo 1 docs use term `crate` quite extensively while protostar and scarb use `package`. I think someone should yield eventually and we should agree on one name, but for the time being, we should at least acknowledge this difference.

Reply:
I understand, I just think it would be far more understandable if you wrote something like "In Cairo 1 docs they usually use the name 'crate' which is basically the same as a package here", why don't we do this?
I mean, "some tools"? How a user may know what you had in mind? What tools? Linux commands? Node.js packages? C++ libs? Cairo, not Cairo, solidity?

Reply:
I can extend the info so it references cairo 1 docs explicitly. However, since `crate` is "official" term, other tools that are not made by us can also use term crate - that's why I didn't specify any tool initially.